save-backend/src/main/kotlin/com/saveourtool/save/backend/service/AgentService.kt

```diff
@@ -23,4 +23,10 @@ class AgentService(private val agentRepository: AgentRepository) {
      */
     internal fun deleteAgentByExecutionIds(executionIds: List<Long>) =
             agentRepository.deleteByExecutionIdIn(executionIds)
+
+    /**
+     * @param containerId
+     * @return ID of [Execution]
+     */
+    internal fun getExecutionId(containerId: String): Long? = agentRepository.findByContainerId(containerId)?.requiredId()
```

Comment:
need to refactor it to be ready to case when a single container can be used for different executions (long running agent)